src/taco-cli/cli/kit.ts

```diff
@@ -427,7 +427,7 @@ class Kit extends commands.TacoCommandBase {
             logger.log(resources.getString("CommandKitSelectStatusUpdatingPlugins"));
         }
 
-        var downloadOptions: Cordova.ICordovaDownloadOptions = { searchpath: "", noregistry: false, usegit: false, cli_variables: {}, browserify: "", link: "", save: true, shrinkwrap: false };
+        var downloadOptions: Cordova.ICordovaDownloadOptions = { searchpath: undefined, noregistry: undefined, usegit: undefined, cli_variables: {}, browserify: undefined, link: undefined, save: true, shrinkwrap: undefined };
```

Comment:
Perhaps we should modify our typings here and make all these undefined entries optional, and simply not specify them here? An object that sets `foo: undefined` is subtly different to an object that does not define `foo`.